Framework/Instamojo.framework/Headers/Instamojo-Bridging-Header.h

```diff
@@ -0,0 +1,16 @@
+//
+//  Instamojo-Bridging-Header.h
+//  Instamojo
+//
+//  Created by Sukanya Raj on 16/02/17.
```

Comment:
Hi Please let me know what it needs to be updated with or this needs to be removed?